dto/group.go

```diff
@@ -19,7 +19,7 @@ type GroupOutputDTO struct {
 	Bills   []BillOutputDTO `json:"bills"`
 }
 
-func (g GroupInputDTO) ToGroup(owner uuid.UUID, members []uuid.UUID) types.Group {
+func (g GroupInputDTO) ToGroup(owner types.User, members []types.User) types.Group {
```

Comment:
If we add the creator ID to the GroupInputDTO the related group service would still get the user from this ID from the database and call the `toGroup` function with the given parameters. 
We would only change the GroupInputDTO and the service would get the user from the given creator ID instead of from the authentication cookie. I don't think `toGroup` has to be changed.

Reply:
Jap, mit der jetzigen Implementierung von ``` AddGroup``` in GroupStorage hast du vollkomend recht.

Folgendes wäre aber interessant zu wissen:
Brauchen wir beim Erstellen der Entität 'Group' in MakeGroup wirklich sowohl Owner (uuid), als auch User zu setzen? 
Ich würde nämlich davon ausgehen, dass das Setzen des primary keys (Owner) ausreichend ist. 

Was ```FirstOrCreate(&groupItem)``` macht, ist einen neuen Eintrag in die groups Tabelle hinzufügen, wo in der "owner" Spalte die Owner/Creator ID eingetragen wird. Es werden dafür sonst keine Infos über User benötigt und in der "users" Tabelle selbst wird auch nichts geschrieben.

Es würde die ganze Logik etwas vereinfachen wenn das wirklich so funktioniert wie ich es mir vorstelle. Ob gorm das intern so auf die Reihe kriegt muss mach noch überprüfen. Können wir aber auch in einem anderen Issue machen.

Reply:
You are right! It is enough to store the owner of the group as an uuid (its foreignkey). Gorm automatic connects the foreign key to the correct user. Therefore, `ToGroup` does not require a types.User parameter, an uuid is sufficient.

I will revise the code.

When working on #28, the owner parameter can completely be removed.